repos/system_upgrade/el7toel8/libraries/config/__init__.py

```diff
@@ -0,0 +1,10 @@
+from leapp.libraries.stdlib import api
+
+
+def get_env_var_value(name):
```

Comment:
Why not just get_env with an or value and default value None as fallback?

Reply:
like this?